File: JavaScript_Courses/JavaScript_Basics/JS_BP_2022_10/01_First_Steps_In_Coding_Exercise/04_Vacation_Books_List.js
function vacationBooks(input) {
    let totalPages = input[0];  // 212
    let pagesPerHour = Number(input[1]);  // 20
    let days = Number(input[2]); // 2

    let totalHours = totalPages / pagesPerHour;

    let hoursPerDay = totalHours / days;

    console.log(hoursPerDay);  // 'log' + Tab
}


vacationBooks([212, 20, "2"]);
// vacationBooks(["432", "15", "4"]);

File: Python_Courses/Python_Basics/Python_PB_2025_03/01_First_Steps_in_Coding_Exercise/04_Vacation_books_list.py
# 1. E
total_pages = int(input())
pages_per_hour = int(input())
days = int(input())

# 2. T
total_hours_needed = total_pages // pages_per_hour
hours_per_day = total_hours_needed / days

# 3. L
print(int(hours_per_day))  # int(7.0) == 7
